BASE/src/Microsoft.ApplicationInsights/Channel/Transmission.cs

```diff
@@ -133,6 +134,47 @@ public ICollection<ITelemetry> TelemetryItems
             get; private set;
         }
 
+        /// <summary>
+        /// Gets a value indicating whether flush is async.
```

Comment:
this comment seems incorrect. Bool indicates whether flush task is completed or not.